apps/chat/src/components/Chat/ConversationSettings.tsx

```diff
@@ -183,10 +183,13 @@ export const ConversationSettings = ({
         <div className="shrink bg-layer-2 px-3 py-4 md:px-6">
           <ConversationSettingsModel
             conversation={conversation}
-            modelId={model?.reference ?? recentModelsIds[0]}
-            unavailableModelId={
-              !model?.id && !isPseudoModel(modelId) ? modelId : undefined
+            modelId={
+              model?.reference ??
+              (!model?.id && !isPseudoModel(modelId)
+                ? modelId
+                : recentModelsIds[0])
```

Comment:
it looks like `recentModelsIds[0]` will be used only if model is undefined and it's pseudomodel... strange